qemu/tests/cfg/virtio_driver_sign_check.cfg

```diff
@@ -1,16 +1,88 @@
+            arch_dir = 64
+        os_dirs = Win2008 2k8
+    Win2008..r2:
+        x86_64:
+            floppy_name = isos/windows/virtio-win-latest-signed-el7_amd64.vfd
+            signtool = "WIN_UTILS:\signtool\signtool-HCK\Signtool_amd64_HCK.exe" 
+            arch_dir = 64
+        os_dirs = Win7 2k8R2
+    Win7:
+        i386, i686:
+            floppy_name = isos/windows/virtio-win-latest-signed-el7_x86.vfd
+            signtool = "WIN_UTILS:\signtool\signtool-HCK\Signtool_x86_HCK.exe"
+            arch_dir = 86
+        x86_64:
+            floppy_name = isos/windows/virtio-win-latest-signed-el7_amd64.vfd
+            signtool = "WIN_UTILS:\signtool\signtool-HCK\Signtool_amd64_HCK.exe" 
+            arch_dir = 64
+        os_dirs = Win7 w7
+    Win8..0:
+        i386, i686:
+            floppy_name = isos/windows/virtio-win-latest-signed-el7_x86.vfd
+            signtool = "WIN_UTILS:\signtool\signtool-HCK\Signtool_x86_HCK.exe"
+            arch_dir = 86
+        x86_64:
+            floppy_name = isos/windows/virtio-win-latest-signed-el7_amd64.vfd
+            signtool = "WIN_UTILS:\signtool\signtool-HCK\Signtool_amd64_HCK.exe" 
+            arch_dir = 64
+        os_dirs = Win8 w8
+    Win8..1:
+        i386, i686:
+            floppy_name = isos/windows/virtio-win-latest-signed-el7_x86.vfd
+            signtool = "WIN_UTILS:\signtool\signtool-HCK\Signtool_x86_HCK.exe"
+            arch_dir = 86
+        x86_64:
+            floppy_name = isos/windows/virtio-win-latest-signed-el7_amd64.vfd
+            signtool = "WIN_UTILS:\signtool\signtool-HCK\Signtool_amd64_HCK.exe" 
+            arch_dir = 64
+        os_dirs = Win8.1 w8.1
+    Win2012..r1:
+        x86_64:
+            floppy_name = isos/windows/virtio-win-latest-signed-el7_amd64.vfd
+            signtool = "WIN_UTILS:\signtool\signtool-HCK\Signtool_amd64_HCK.exe" 
+            arch_dir = 64
+        os_dirs = Win8 2k12
+    Win2012..r2:
+        x86_64:
+            floppy_name = isos/windows/virtio-win-latest-signed-el7_amd64.vfd
+            signtool = "WIN_UTILS:\signtool\signtool-HCK\Signtool_amd64_HCK.exe" 
+            arch_dir = 64
+        os_dirs = Win8 2k12R2
+    Win10:
+        i386, i686:
+            floppy_name = isos/windows/virtio-win-latest-signed-el7_x86.vfd
+            signtool = "WIN_UTILS:\signtool\signtool-HLK\Signtool_x86_HLK.exe"
+            arch_dir = 86
+        x86_64:
+            floppy_name = isos/windows/virtio-win-latest-signed-el7_amd64.vfd
+            signtool = "WIN_UTILS:\signtool\signtool-HLK\Signtool_amd64_HLK.exe"
+            arch_dir = 64
+        os_dirs = Win10 w10
+    Win2016:
+        x86_64:
+            floppy_name = isos/windows/virtio-win-latest-signed-el7_amd64.vfd
+            signtool = "WIN_UTILS:\signtool\signtool-HLK\Signtool_amd64_HLK.exe"
+            arch_dir = 64
+        os_dirs = Win10 2k16
+    signtool_cmd = ${signtool} verify /v /kp /c %s %s
```

Comment:
floppy_name and signtool parameters of each widows platforms are kind of duplicated,please re-arrange them.

Reply:
so is arch_dir

Reply:
ok, will re-arrange it, thanks~